Makefile

```diff
@@ -1,6 +1,9 @@
 BUILD_CHANNEL?=local
 TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
 PATH_WITH_TOOLS="`pwd`/$(TOOL_BIN):${PATH}"
+GIT_REVISION = $(shell git rev-parse HEAD | tr -d '\n')
+TAG_VERSION?=$(shell etc/tag_version.sh)
+LDFLAGSOPTS = -ldflags "-X 'main.Version=${TAG_VERSION}' -X 'main.GitRevision=${GIT_REVISION}'"
```

Comment:
I changed this from LDFLAGS as CMAKE was acting on this ENV var during C++ compilation & failing to link carto  C++ code.